_docs/guides/bookinfo.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Bookinfo on Kubernetes
```

Comment:
Let's keep "on Kubernetes" or drop the consul one

Reply:
Direction proposed by this PR is the latter. This guide just describes the sample and gives basic run/stop instructions for all platforms.

Reply:
Not sure it will flow well to not specify the platform but if you can pull it off, sounds good

removal of "Bookinfo with Consul" should be in this same PR then